src/content/docs/ru/guides/imports.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: Импорты
+description: Узнайте, как импортировать разные типы контента в Astro.
+i18nReady: true
+---
+import RecipeLinks from "~/components/RecipeLinks.astro";
+import ReadMore from '~/components/ReadMore.astro'
+
+Astro поддерживает большинство статических ресурсов без необходимости настройки. Вы можете использовать оператор `import` в любом месте вашего JavaScript-кода (включая Astro frontmatter), и Astro включит оптимизированную копию этого ресурса в итоговую сборку. `@import` также поддерживается в CSS и `<style>` тегах.
+
+## Поддерживаемые типы файлов
+
+Astro по умолчанию поддерживает следующие типы файлов:
+
+- Компоненты Astro  (`.astro`)
+- Markdown (`.md`, `.markdown`, etc.)
+- JavaScript (`.js`, `.mjs`)
+- TypeScript (`.ts`)
+- Пакеты NPM
+- JSON (`.json`)
+- CSS (`.css`)
+- CSS-модули (`.module.css`)
+- Изображения и ресурсы (`.svg`, `.jpg`, `.png`, etc.)
+
+Вы также можете расширить Astro для поддержки других [UI фреймворков](/ru/guides/framework-components/) таких как React, Svelte и Vue. Можно установить [интеграцию Astro MDX](/ru/guides/integrations-guide/mdx/) и использовать .mdx файлы в вашем проекте.
+
+### Файлы в директории `public/`
+
+Вы можете поместить любой статический ресурс в [директорию `public/`](/ru/basics/project-structure/#public) вашего проекта, и Astro скопирует его в итоговую сборку без изменений. Файлы в `public/` не обрабатываются Astro, так что поддерживается любой тип файла. К файлу `public/` можно обращаться по URL-адресу напрямую в HTML-шаблонах.
+
+## Операторы импорта
+
+Astro использует ESM, те же [`import`](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/import#syntax) и [`export`](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/export) операторы, которые поддерживаются в браузерах.
+
+### JavaScript
+
+```js
+import { getUser } from './user.js';
+```
+
+JavaScript можно импортировать с помощью обычного синтаксиса import & export.
+
+:::note[Импорт файлов JSX]
+
+Для рендеринга файлов JSX/TSX требуется соответствующий [UI фреймворк](/ru/guides/framework-components/) ([React](/ru/guides/integrations-guide/react/), [Preact](/ru/guides/integrations-guide/preact/), или [Solid](/ru/guides/integrations-guide/solid-js/)).
+Используйте расширения `.jsx`/`.tsx` где это необходимо, так как Astro не поддерживает JSX в файлах `.js`/`.ts`.
+
+:::
+
+### TypeScript
+
+```js
+import { getUser } from './user';
+import type { UserType } from './user';
+```
+
+Astro включает встроенную поддержку TypeScript. Вы можете импортировать `.ts` и `.tsx` файлы напрямую в ваш проект Astro и даже писать TypeScript-код прямо внутри [скрипта компонента Astro](/ru/basics/astro-components/#the-component-script) и любых [тегах скриптов](/ru/guides/client-side-scripts/).
```

Comment:
```suggestion
Astro включает встроенную поддержку [TypeScript](https://www.typescriptlang.org/). Вы можете импортировать `.ts` и `.tsx` файлы напрямую в ваш проект Astro и даже писать TypeScript-код прямо внутри [скрипта компонента Astro](/ru/basics/astro-components/#скрипт-компонента) и любых [тегах скриптов](/ru/guides/client-side-scripts/).
```

Fixed invalid links.

If a Russian-language page for the TypeScript site exists, please let me know! I think you can fix it with that.